Maple2.Server.Game/Model/Field/Entity/FieldMobSpawn.cs

```diff
@@ -59,14 +67,88 @@ public void Despawn(int objectId) {
         }
     }
 
+    private List<Vector3> GetRandomSpawns(int count) {
+        List<Vector3> spawnsPicked = new();
+        Vector3[] spawnsRemaining = new Vector3[validSpawns.Count];
+
+        validSpawns.CopyTo(spawnsRemaining);
+
+        int selectSpawns = int.Min(count, validSpawns.Count);
+        int remainder = count - selectSpawns;
+
+        for (int i = 0; i < selectSpawns; ++i) {
+            int picked = Random.Shared.Next(0, spawnsRemaining.Length - i);
```

Comment:
Redundant to declare 0.

Reply:
Can you clarify please? Which 0 here?

Reply:
`Random.Shared.Next(0, spawnsRemaining.Length - i);` is equal to Random.Shared.Next(spawnsRemaining.Length - i);